node-fs-writefile/write-rand.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs');
+
+const randomNum = Math.random();
+const stringNum = randomNum.toString();
+
+const data = new Uint8Array(Buffer.from(stringNum));
+
+fs.writeFile('random.txt', data, err => {
+  if (err) throw err;
+});
```

Comment:
You don't need to create a byte array to write UTF-8 strings to files, the `writeFile` method accepts strings directly: https://nodejs.org/api/fs.html#fswritefilefile-data-options-callback

Reply:
So i should be able to pass in the const 'stringNum' in place of data?

Reply:
i added another commit getting rid of the byte array